docs/Database.md

```diff
@@ -28,7 +29,8 @@ and are also listed here:
 
 Attribute Name | Description
 ---|---
-`github_team_name` | `String`; The team's github name
+`gh_team_id` | `String`; The team's Github ID
+`github_team_name` | `String`; The team's Github name
```

Comment:
I don't like the inconsistency; we have:
* `gh_team_id`
* `github_team_name`
* `guid`
I'd prefer if these all followed a set naming convention (and, if possible, one that included the Slack ID as well).

Reply:
I think leaving slack ID labelled `slack_id` should be enough, at least for the length. Naming it `s_id` would be a bit confusing. I have gone and renamed all instances of `github...` and `g...` to `gh...`.

Reply:
nvm; renamed all instances to `github_`